test/e2e/fixtures/fixtures.go

```diff
@@ -0,0 +1,984 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package fixtures
+
+import (
+	greenhousev1alpha1 "github.com/cloudoperators/greenhouse/pkg/apis/greenhouse/v1alpha1"
+	"github.com/cloudoperators/greenhouse/pkg/test"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var NginxPluginDefinition = &greenhousev1alpha1.PluginDefinition{
+	TypeMeta: metav1.TypeMeta{
+		Kind:       "PluginDefinition",
+		APIVersion: greenhousev1alpha1.GroupVersion.String(),
+	},
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "nginx-18.1.7",
+		Namespace: test.TestNamespace,
+	},
+	Spec: greenhousev1alpha1.PluginDefinitionSpec{
+		Description: "nginx",
+		HelmChart: &greenhousev1alpha1.HelmChartReference{
+			Name:    "nginx",
+			Version: "18.1.7",
+		},
+		Options: []greenhousev1alpha1.PluginOption{
```

Comment:
Hehe quite a long list of options. I think we can reduce this. Maybe start by keeping  `autoscaling.enabled`, `autoscaling.minReplicas`, and `autoscaling.maxReplicas`
Then we can use these also to check that updates to the Plugin lead to remote resources being updated.